tasks/main.yml

```diff
@@ -6,7 +6,7 @@
   changed_when: false
 
 - name: Download get-pip.py
-  get_url: url=https://bootstrap.pypa.io/get-pip.py
+  get_url: url=https://bootstrap.pypa.io{{ "/" if get_pip_version == "latest" else "/" + get_pip_version + "/"}}get-pip.py
```

Comment:
Slight nitpick, but I think that the approach is a little bit more readable because it gets rid of the confusion around where the slashes will render:

```diff
diff --git a/tasks/main.yml b/tasks/main.yml
index b3e36ac..04e6e50 100644
--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -6,7 +6,7 @@
   changed_when: false
 
 - name: Download get-pip.py
-  get_url: url=https://bootstrap.pypa.io{{ "/" if get_pip_version == "latest" else "/" + get_pip_version + "/"}}get-pip.py
+  get_url: url="https://bootstrap.pypa.io/{{ (get_pip_version == 'latest') | ternary('', get_pip_version) }}/get-pip.py"
            dest=/tmp/get-pip.py
   when: pip_version_output | failed or not pip_version_output.stdout | search(pip_version)
```

Reply:
I don't think this is a nitpick. I was trying to figure out an elegant way to do this conditional. Now I am aware of the `ternary()` method!